website/profile/utils.py

```diff
@@ -107,6 +111,11 @@ def serialize_contributors(contribs, node, **kwargs):
         for contrib in contribs
     ]
 
+def serialize_visible_contributors(node):
+    # This is optimized when node has .include('contributor__user__guids')
+    return [
+        serialize_user(c, node) for c in node.contributor_set.all() if c.visible
```

Comment:
Not a big deal for the 80%. It's probably not worth the time we talked about it. Feel free to just leave it as written.